components/globalProblem/GlobalProblemOrganizationsList.tsx

```
@@ -1,139 +1,170 @@
       org.industry?.toLowerCase().includes(search.toLowerCase()) ||
-      org.mission?.toLowerCase().includes(search.toLowerCase()) ||
       org.description?.toLowerCase().includes(search.toLowerCase())
     )
-    setFilteredOrganizations(filtered)
-  }, [search, organizations])
+    .sort((a, b) => {
+      switch (sortBy) {
+        case "name":
+          return a.name.localeCompare(b.name)
+        case "focusLevel":
+          const aWeight = a.focusLevel ? FOCUS_LEVEL_WEIGHTS[a.focusLevel] : -1
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)